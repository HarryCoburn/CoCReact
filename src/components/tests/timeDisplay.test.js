import React from "react";
import { Provider } from "react-redux";
import renderer from "react-test-renderer";
import configureStore from "redux-mock-store";
import TimeDisplay from "../timeDisplay";

const mockStore = configureStore([]);

describe("Time Display Component", () => {
  let store;
  let component;

  beforeEach(() => {
    store = mockStore({
      time: {
        day: "0",
        minute: "12",
        hour: "5"
      }
    });

    component = renderer.create(
      <Provider store={store}>
        <TimeDisplay />
      </Provider>
    );
  });

  it("should match the snapshot", () => {
    expect(component.toJSON()).toMatchSnapshot();
  });
});
