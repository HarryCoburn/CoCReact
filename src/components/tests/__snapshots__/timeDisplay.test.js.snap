// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Time Display Component should match the snapshot 1`] = `
<div
  className="timeDisplay"
>
  <p>
    Day#: 
    0
    <br />
    Time: 
    5
    :
    12
  </p>
</div>
`;
